apollo-router/src/json_ext.rs

```diff
@@ -468,6 +474,42 @@ impl ValueExt for Value {
                     typename == maybe_type || schema.is_subtype(maybe_type, typename)
                 })
     }
+
+    fn to_ast(&self) -> apollo_compiler::ast::Value {
+        match self {
+            Value::Null => apollo_compiler::ast::Value::Null,
+            Value::Bool(b) => apollo_compiler::ast::Value::Boolean(*b),
+            Value::Number(n) => {
+                if n.is_f64() {
```

Comment:
Is there a better way to do this conversion? This inner `n` is private so it's not so easy to cleanly convert it to the underlying numeric type.

Reply:
An alternative is:
```
            Value::Number(n) if n.is_f64() => {
                apollo_compiler::ast::Value::Float(n.as_f64().expect("is float").into())
            }

            Value::Number(n) => {
                apollo_compiler::ast::Value::Int((n.as_i64().expect("is int") as i32).into())
            }
```